modules/post/multi/gather/jboss_gather.rb

```diff
@@ -0,0 +1,266 @@
+require 'msf/core'
+require 'nokogiri'
+
+class MetasploitModule < Msf::Post
+    include Msf::Post::File
+    include Msf::Post::Linux::System
+
+    def initialize(info={})
+    super(update_info(info,
+        'Name'          => 'Jboss Credential Collector',
+        'Description'   => %q{
+          This module can be used to extract the Jboss admin passwords for version 4,5 and 6.
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Koen Riepe (koen.riepe@fox-it.com)' ],
+        'Platform'      => [ 'linux', 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
+    ))
+    end
+
+    def report_creds(user, pass, port)
```

Comment:
Passing `port` to `report_creds` seems super weird. Take a look at a similar module and steal the `report_creds` method from it instead of writing your own.